### ISSUE TITLE
Enhance CI support of commit messages

### ISSUE THREAD

**Issue by plroebuck** [netlify/ask-netlify#27]
## My question in a little more detail
From the [Netlify documentation](https://www.netlify.com/docs/continuous-deployment/#skipping-a-deploy)...

> Sometimes, you may want to push commits to your production branch without triggering a deploy on Netlify. To do this, add `[skip ci]` anywhere in the Git commit message.

Request two minor enhancements:
1. Add support for the converse `[ci skip]` -- nobody can remember which way it's supposed to be specified
1. Add support for `[skip netlify]` -- this will save wasted processing time when changes will not affect the documentation/website

## Additional comments for Netlify
Sorry if this isn't the right place for this enhancement request, but it's not really apparent where this issue should have been submitted.

See also:
* bors-ng/bors-ng#522
* travis-ci/travis-ci#911
* travis-ci/travis-ci#5032
* [TravisCI skip directives](https://docs.travis-ci.com/user/customizing-the-build/#skipping-a-build)
* [AppVeyor skip directives](https://www.appveyor.com/docs/how-to/filtering-commits/#skip-directive-in-commit-message)
* [CircleCI skip directives](https://circleci.com/docs/2.0/skip-build/#skipping-a-build)

**Comment by philhawksworth** [comment]
Thanks for this @plroebuck. Interesting suggestions. We'll raise them with the appropriate people and capture the request.

**Comment by plroebuck** [comment]
So... closing this means what? Was it implemented?

**Comment by j-f1** [comment]
@plroebuck Probably means it’ll be covered in the next episode.

**Comment by plroebuck** [comment]
episode === release? How will I know when I can start using the functionality?

**Comment by j-f1** [comment]
They’re going to cover it in their YouTube show, and they’ll probably talk about their plans there.

**Comment by philhawksworth** [comment]
Ah... sorry @plroebuck I should clarify this with a standard message when we close an issue on here. This repo is purely for the submission of questions for the Ask Netlify series of podcasts. We close the issues when we have recorded an answer.

Expect to see your question discussed in episode 3, which we recently recorded.

For a little extra info now.... this request makes a lot of sense. I' created an internal issue for our product team to triage and plan against other priorities. I can't promise when, but it is a good suggestion which I have passed along to the team.

More details in the episode when it is published. Thanks for the question!
